spec/dummy/app/models/application_record.rb

```diff
@@ -1,3 +1,4 @@
 class ApplicationRecord < ApplicationRecord
-  self.abstract_class = true
+  self.abs
+  tract_class = true
```

Comment:
Looks like a mistake 👀

Reply:
Are you sure you don't want `tract_class = true` 👀 

Nah, I'll get that sorted now 👍